### PR TITLE
WIC/ECE Treatment 2 - apply

### PR DESCRIPTION
**Pull request by sree-cfa**

### CODE REVIEW

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -19,6 +19,8 @@ general.check-all-that-apply=Check all that apply.
 general.you=(You)
 general.none-of-the-above=None of the above
 general.good-news=Good news!
+general.i-dont-know=I don't know
+general.not-interested=No, I\u2019m not interested
```
Comment (analoo) [inline src/main/resources/messages.properties:23]: Just curious why line 22 uses an apostrophe and line 23 uses unicode

File: src/main/resources/messages.properties [hunk 2]
```diff
@@ -1066,51 +1068,49 @@ parish.header=Which parish do you live in right now?
...
 wic.would-you-like-share-data=Would you like to share your data with WIC to see if you qualify?
-share-wic-button=Share my data with WIC
-
-you-can-finish-up-2=You can finish up your SNAP application, and receive next steps for all applications at the end.
 
-# WIC / ECE
 nola-ece-links.title=ECE link
-nola-ece-links.if-youre-approved=If you\u2019re approved for SNAP, you\u2019re also eligible for free childcare and pre-K.
-nola-ece-links.subheader=These programs can:<ul class="list--bulleted"> <li>Provide free childcare and education for children from birth through age 4</li> <li>Provide free pre-K in public or private school for 4-year-olds</li></ul>
+nola-ece.if-youre-approved=If you\u2019re approved for SNAP, you\u2019re also eligible for free childcare and pre-K.
```
Comment (analoo) [inline src/main/resources/messages.properties:1112]: Again here with the unicode

File: src/test/java/org/ladocuploader/app/journeys/LaDigitalAssisterFlowJourneyTest.java [hunk 3]
```diff
@@ -653,6 +653,15 @@ void fullDigitalAssisterFlow() {
       testPage.clickButton("Yes");
       assertThat(testPage.getTitle()).isEqualTo("WIC link");
       testPage.clickButton("Yes");
+    } else if ("ECE apply".equals(title)) {
```
Comment (analoo) [inline src/test/java/org/ladocuploader/app/journeys/LaDigitalAssisterFlowJourneyTest.java:656]: So cool! I didn't realize we could add else statements in the testing flow.

**Review by analoo — APPROVED** [review]
Looks good to me. Did not do a QA.

File: src/main/resources/messages.properties (hunk 1 above)
Comment (sree-cfa) [reply src/main/resources/messages.properties:23]: Sometimes Figma c/p has a unicode apostrophe 🤷

**Pull request merged**